src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -522,19 +522,6 @@ def _deploy_workflow(self, step_name: str, settings):
         self._install_state.jobs[step_name] = str(new_job.job_id)
         return None
 
-    def _upload_wheel(self, prompts: Prompts):
-        with self._wheels:
-            try:
-                self._wheels.upload_to_dbfs()
-            except PermissionDenied as err:
-                if not prompts:
-                    raise RuntimeWarning("no Prompts instance found") from err
-                logger.warning(f"Uploading wheel file to DBFS failed, DBFS is probably write protected. {err}")
```

Comment:
this is still required for write-protected DBFS. 

https://github.com/databrickslabs/ucx/issues/1098 is about removing dbfs uploads from other places.